### DISCUSSION TITLE
Cloudflare Domain

### DISCUSSION THREAD

**Discussion by Zakir-Muhammad** [community/community#123048] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I recently transferred my domain to Cloudflare. Previously, my GitHub pages site was functioning correctly, but since I switched to Cloudflare, the browser keeps loading and never shows my page. The URL is https://www.zakirm.com. Any idea what I did wrong? Do my DNS records look fine?

<img width="633" alt="image" src="https://github.com/community/community/assets/84191340/ccb04778-9537-4cc2-8f03-63a3c8f72de6">

**Comment by LiteBrite82** [comment]
Looks like it may be working now; the page loaded for me.

**Reply by Zakir-Muhammad** [reply, marked as answer]
Yes, it's working now. Seems like I had to make a commit for the deployment to start working again.

**Reply by LiteBrite82** [reply]
Sweet! I'm glad it worked! Nice profile by the way! Good luck on the internship hunt!

**Comment by Zakir-Muhammad** [comment]
It started working after I made a commit.

**Comment by msi-end** [comment]
Here's how you can check your DNS records in Cloudflare:

**Login to Cloudflare:** Go to Cloudflare's website and log in to your account.
**Select your domain:** Once logged in, select the domain zakirm.com from your Cloudflare dashboard.
**Check DNS records:** Look for the DNS settings or DNS management section within Cloudflare. Here, you should see a list of DNS records associated with your domain.
Now, let's discuss what to look for in your DNS records:

**A Record for the root domain** : Ensure that you have an A record pointing to the IP address of your GitHub Pages site. This record should have @ or your root domain (zakirm.com) in the Name field, and the IP address of your GitHub Pages server in the Content field.
**CNAME Record for www subdomain (optional):** If you want your site to be accessible via www.zakirm.com, make sure you have a CNAME record pointing to your GitHub Pages domain. The Name field should have www, and the Content field should be set to your GitHub Pages domain (typically <username>.github.io).
**SSL/TLS settings:** Check your SSL/TLS settings in Cloudflare to ensure that HTTPS is enabled. You should choose the appropriate SSL mode (e.g., Full or Full (strict)) based on your GitHub Pages configuration.
**Cloudflare Proxy Status:** If you want to leverage Cloudflare's performance and security features, ensure that the proxy status (the orange cloud icon) is enabled for the DNS records related to your website.
After verifying your DNS records, give it some time for the changes to propagate. DNS changes can take up to 24 hours to fully propagate worldwide, although it typically happens much faster.
